cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java

```diff
@@ -97,19 +97,19 @@ private String azureError(CloudStorage cloudStorage) {
 
         String auditLocation = getRangerAuditDir(cloudStorage);
         AccountMapping accountMapping = cloudStorage.getAccountMapping();
-        String dataAccessRole = accountMapping.getUserMappings().get("hive");
+        String dataAccessIdentity = accountMapping.getUserMappings().get("hive");
         // There is no ranger in the IDBroker mapping, so we can use solr
-        String rangerAuditRole = accountMapping.getUserMappings().get("solr");
+        String rangerAuditIdentity = accountMapping.getUserMappings().get("solr");
 
         return String.format("Services running on the cluster were unable to write to %s location. " +
                         "This problem usually occurs due to cloud storage permission misconfiguration. " +
                         "Services on the cluster are using Data Access Identity (%s) and Ranger Audit Identity (%s) to write to the " +
                         "Ranger Audit location (%s), therefore please verify that these roles have write access to this location. " +
-                        "During Data Lake cluster creation, CDP Control Plane attaches Data Access Assumer Identity (%s) to the IDBroker VM. " +
-                        "IDBroker will then use it to attach the other managed identities to the IDBroker VM, therefore Data Access Identity (%s) " +
+                        "During Data Lake cluster creation, CDP Control Plane attaches Assumer Identity (%s) to the IDBroker VM. " +
+                        "IDBroker will then use it to attach the other managed identities to the IDBroker VM, therefore Assumer Identity (%s) " +
                         "permissions must, at a minimum, allow to attach the Data Access Identity and Ranger Access Identity. " +
                         "Refer to Cloudera documentation at %s for the required rights.",
-                auditLocation, dataAccessRole, rangerAuditRole, auditLocation, managedIdentity, managedIdentity,
+                auditLocation, dataAccessIdentity, rangerAuditIdentity, auditLocation, managedIdentity, managedIdentity,
```

Comment:
I suggest renaming `managedIdentity` to `assumerIdentity` for consistency.

Reply:
Thanks. Makes sense.